README.md

```diff
@@ -18,7 +18,7 @@ $ dokku apps:create my-litestream-rails
 $ dokku storage:ensure-directory my-litestream-rails--db
 
 # Mount the storage to your app
-$ dokku storage:mount /var/lib/dokku/data/storage/my-litestream-rails--db:/app/db/litestream
+$ dokku storage:mount my-sqlite-application /var/lib/dokku/data/storage/my-sqlite-application--db:/app/db/litestream
```

Comment:
Thanks, great catch we're missing the application name.

But in the docs example the app is called `my-litestream-rails` and the directory `my-litestream-rails--db` so the full command should be `dokku storage:mount my-litestream-rails /var/lib/dokku/data/storage/my-litestream-rails--db:/app/db/litestream`